_layouts/default.html

```diff
@@ -19,12 +19,12 @@
     <script language="javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA" id="_fed_an_ua_tag"></script>
 
     <!-- Google Tag Manager -->
-    <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-KKLQ6F"
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KKLQ6F"
     height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
     <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
     new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
     j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
```

Comment:
I also added a couple of explicit protocols to the Google Tag Manager code. However, you should review that the URLs you're pulling in via Google Tag Manager do the same thing. 

The Google Analytics code your GTM is currently configured to pull in doesn't look inline with the privacy standards that [18F has committed to](https://github.com/18F/analytics-standards#18f-account-snippet), where TLS is enforced and IPs are anonymized. I can't submit a PR to fix that, though.